test/exec-util.ts

```diff
@@ -40,6 +48,7 @@ export function execSnapshot(cmd: string, options?: CallOptions): ExecSnapshot {
 
 const defaultExecResult = { stdout: '', stderr: '' };
 
+// TODO: rename
 export function mockExecAll(
   execFn: ExecMock,
```

Comment:
@zharinov Can we refactor this, so we use the local `exec` value, so tests don't need to pass import and pass the execMock?

Should simplify exec mocks. I think we can call mock on `utils/exec` here too, as we expect it to be mocked anyways when this utils are used.